core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -891,14 +913,22 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
   }
 
   /** Build the union of a list of RDDs. */
-  def union[T: ClassTag](rdds: Seq[RDD[T]]): RDD[T] = new UnionRDD(this, rdds)
+  def union[T: ClassTag](rdds: Seq[RDD[T]]): RDD[T] = {
+    assertNotStopped()
+    new UnionRDD(this, rdds)
+  }
 
   /** Build the union of a list of RDDs passed as variable-length arguments. */
-  def union[T: ClassTag](first: RDD[T], rest: RDD[T]*): RDD[T] =
+  def union[T: ClassTag](first: RDD[T], rest: RDD[T]*): RDD[T] = {
+    assertNotStopped()
     new UnionRDD(this, Seq(first) ++ rest)
+  }
 
   /** Get an RDD that has no partitions or elements. */
-  def emptyRDD[T: ClassTag] = new EmptyRDD[T](this)
+  def emptyRDD[T: ClassTag] = {
```

Comment:
This did _not_ cause an exception in 1.2, so I'll remove this `assertNotStopped` call.